### PR TITLE
Add an option remove

### PR DESCRIPTION
**Pull request by dmitrykiselyov**
Allow a user to define a function that takes each comment and depending
on the conditions returns true to remove a comment or false to keep a
comment.

For example: remove: function(comment) { return comment[0] == "@"; }
removes all /*@ comments */

Closes-Issue: #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,6 +30,28 @@ all regular comments (`/* comment */`) and preserve comments marked as important
 
 #### options
 
+##### remove
+
+Type: `function`
+Return: `boolean`
+Variable: `comment` contain a comment withot `/**/`
```
Comment (ben-eb) [inline README.md]: *without*

File: README.md [hunk 2]
```diff
@@ -30,6 +30,28 @@ all regular comments (`/* comment */`) and preserve comments marked as important
 
 #### options
 
+##### remove
```
Comment (ben-eb) [inline README.md]:
```
##### remove(function)
```

File: README.md [hunk 3]
```diff
@@ -30,6 +30,28 @@ all regular comments (`/* comment */`) and preserve comments marked as important
...
+
+Type: `function`
+Return: `boolean`
+Variable: `comment` contain a comment withot `/**/`
+
+The function takes each `comment` and depending on the conditions returns
+`true` to remove a comment or `false` to keep a comment.
+
```
Comment (ben-eb) [inline README.md]: For each comment, return `true` to remove, or `false` to keep the comment.

File: README.md [hunk 4]
```diff
@@ -30,6 +30,28 @@ all regular comments (`/* comment */`) and preserve comments marked as important
...
+```
+
+```js
+var css = '/* headings *//*@ h1 */h1{margin:0 auto}/*@ h2 */h2{color:red}';
+console.log(postcss(comments({
+    remove: function(comment) { return comment[0] == "@"; }
+})).process(css).css);
+//=> /* headings */h1{margin:0 auto}h2{color:red}
+```
+**NOTE:** If you use the `remove` other options will be not available!
```
Comment (ben-eb) [inline README.md]: If you use the `remove` function other options will not be available.

**Comment by ben-eb** [comment]
Looks good to me, can you fix the documentation according to my comments and squash the commits down? Thanks. :+1:

**Comment by dmitrykiselyov** [comment]
Ok, I need to figure out how to squash the commits down :smile:

**Comment by ben-eb** [comment]
There's a quick guide here: http://blog.steveklabnik.com/posts/2012-11-08-how-to-squash-commits-in-a-github-pull-request

**Comment by dmitrykiselyov** [comment]
Thank you for the link.
I hope I did it right. It was my first rebase :smile:

**Comment by ben-eb** [comment]
Looks great! Thanks for your contribution. :smiley:

**Pull request merged**